boards/ti/sk_am64/doc/index.rst

```diff
@@ -4,12 +4,20 @@ Overview
 ********
 
 The SK-AM64 board configuration is used by Zephyr applications that run on
-the TI AM64x platform. The board configuration provides support for the ARM
-Cortex-M4F MCU core and the following features:
+the TI AM64x platform.
+
+The ``sk_am64/am6442/m4`` board configuration provides support for the ARM
+Cortex-M4F MCU core with the following features:
 
 - Nested Vector Interrupt Controller (NVIC)
 - System Tick System Clock (SYSTICK)
 
+The ``sk_am64/am6442/r5f0_0`` board configuration provides support for the ARM
+Cortex-R5F MCU core with the following features:
+
+- Interrupt Controller (VIM)
+- System Tick System Clock (DMTimer)
+
```

Comment:
Can you update the documentation and the relevant .yml files to reflect the features supported?

Reply:
Thanks, this makes perfect sense.